docs/_embeds/apis/rust/consumer-auto-offsets.rs

```diff
@@ -0,0 +1,34 @@
+use async_std::stream::StreamExt;
+
+use fluvio::{
+    consumer::{ConsumerConfigExtBuilder, OffsetManagementStrategy},
+    Fluvio, Offset,
+};
+  
+const TOPIC_NAME: &str = "hello-rust";
+const PARTITION_NUM: u32 = 0;
+const CONSUMER_OFFSET: &str = "consumer-auto";
+
+#[async_std::main]
+async fn main() {
+    // Connect to Fluvio cluster
+    let fluvio = Fluvio::connect().await.unwrap();
+
+    // Consume last record from topic
+    let config = ConsumerConfigExtBuilder::default()
+        .topic(TOPIC_NAME)
+        .partition(PARTITION_NUM)
+        .offset_start(Offset::end())
+        .offset_consumer(CONSUMER_OFFSET.to_string())
+        .offset_strategy(OffsetManagementStrategy::Auto)
+        .build()
+        .unwrap();
+
+
+    // Create consumer & stream one record
+    let mut stream = fluvio.consumer_with_config(config).await.unwrap();
```

Comment:
```suggestion
    let mut stream = fluvio.consumer_with_config(config).await?;
```

Reply:
these are examples, no ned for error mapping.

Reply:
unwrap is ok for writing tests and few corner cases.    Having unwrap makes code hard to read;

Reply:
Moved to expect